### PR TITLE
move cluster and node test to _test package

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**:  this is the fourth step to moving unit tests from handler package to handler_test package

**Special notes for your reviewer**:
The issue for this task: #2359 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/cluster_test.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
```
Comment (p0lyn0mial) [inline api/pkg/handler/cluster_test.go:12]: shouldn't we change the package name for this file to `handler_test` ?

File: api/pkg/handler/node_v1_test.go [hunk 2]
```diff
@@ -12,8 +12,10 @@ import (
 	"time"
```
Comment (p0lyn0mial) [inline api/pkg/handler/node_v1_test.go:13]: same as https://github.com/kubermatic/kubermatic/pull/2611/files#r247130307

Comment (zreigz) [reply api/pkg/handler/node_v1_test.go:13]: yes, it was the purpose, please hold on

**Comment by zreigz** [comment]
@p0lyn0mial PTAL

File: api/pkg/handler/cluster_test.go (hunk 1 above)
Comment (zreigz) [reply api/pkg/handler/cluster_test.go:12]: done

File: api/pkg/handler/node_v1_test.go (hunk 2 above)
Comment (zreigz) [reply api/pkg/handler/node_v1_test.go:13]: fixed

**Pull request merged**